### PR TITLE
Add Stack info links

### LINKED ISSUE #550: Link to Stack docs in learning section
#### Summary

Add links to The Things Stack documentation for some sections in the learning section.

#### Why do we need this ?

To hopefully guide users even better to the Stack docs when they are running into problems.

#### What is already there? What do you see now?

No links in the learning section to the Stack docs.

#### What is missing? What do you want to see?

An info box at some of the sections to link to the Stack docs for more information.

#### How do you propose to document this?

By adding the links to the learning section.

#### Can you do this yourself and submit a Pull Request?

Yes.

### PR DESCRIPTION
**Pull request by LDannijs**
#### Summary

Add some info sections with links to the Stack docs. Closes #550 

#### Changes

- add info boxes to some of the sections

#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/lorawan/adaptive-data-rate/_index.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Mobile end devices should be able to detect when they are stationary for a longe
 
 ## ADR in The Things Stack
 
+{{< info >}}
+Check [The Things Stack documentation](https://www.thethingsindustries.com/docs/reference/adr/#how-adr-works) for more specifics on how ADR works on the Stack.
```
Comment (nejraselimovic) [inline doc/content/lorawan/adaptive-data-rate/_index.md]:
```suggestion
Check [{{% tts %}} documentation](https://www.thethingsindustries.com/docs/reference/adr/#how-adr-works) for more specifics on how ADR works on the {{% tts %}}.
```

File: doc/content/lorawan/end-device-activation/index.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Every end device must be registered with a network before sending and receiving
 
 The join procedure for LoRaWAN 1.0.x and 1.1 is slightly different. The following two sections describe the join procedure for LoRaWAN 1.0.x and 1.1 separately.
 
+{{< info >}}
+For more information on the differences between OTAA and ABP, check the [The Things Stack documentation](https://www.thethingsindustries.com/docs/the-things-stack/concepts/data-formats/).
```
Comment (nejraselimovic) [inline doc/content/lorawan/end-device-activation/index.md]:
```suggestion
For more information on the differences between OTAA and ABP, check the [{{% tts %}} documentation](https://www.thethingsindustries.com/docs/the-things-stack/concepts/data-formats/).
```

File: doc/content/lorawan/message-types/index.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ images:
 - payload.png
 ---
 
+{{< info >}}
+For information on The Things Stack's specific data formats, check [The Things Stack documentation](https://www.thethingsindustries.com/docs/the-things-stack/concepts/data-formats/).
```
Comment (nejraselimovic) [inline doc/content/lorawan/message-types/index.md]:
```suggestion
For information on {{% tts %}} specific data formats, check [{{% tts %}} documentation](https://www.thethingsindustries.com/docs/the-things-stack/concepts/data-formats/).
```

File: doc/content/lorawan/regional-parameters/_index.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ section: The Things Fundamentals
 weight: 40
 ---
 
+{{< info >}}
+For information on the specific versions of the Regional Parameters check [The Things Stack documentation](https://www.thethingsindustries.com/docs/the-things-stack/concepts/spec-regional-parameters/).
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-parameters/_index.md]:
```suggestion
For information on the specific versions of the Regional Parameters check [{{% tts %}} documentation](https://www.thethingsindustries.com/docs/the-things-stack/concepts/spec-regional-parameters/).
```

**Review by nejraselimovic — CHANGES_REQUESTED** [review]
hey @LDannijs, sorry for not seeing this before, reviewers weren't assigned. Just fix TTS tags and we can merge this

**Comment by LDannijs** [comment]
Ehmm it seems there is no tts shortcode for The Things Network docs 😅

**Comment by LDannijs** [comment]
Manually added it by referencing the Stack docs one

**Review by nejraselimovic — APPROVED** [review]
Awesome, thanks!

**Pull request merged**